README.md

```diff
@@ -1,12 +1,18 @@
 The 18F Method Cards (“18F Methods”) describe how 18F puts human-centered design into practice. While this is primarily maintained as an internal resource, we hope it can help *everyone* adopt the methods of human-centered design.
 
-### Why methods?
+## 18F Methods mission and vision
+In late 2016, we articulated a vision and mission for the project that focus our efforts on meeting the needs of government users:
 
-In order to function well within cross-functional teams, designers need to know a few things: which methods they might choose from, why one particular method makes more sense than another at any given moment, and, once they’ve picked a method, how to actually execute it. 18F Methods collects this essential information as a series of cards. In practice, we’ve found the Methods can provide folks with a gateway into our work and build internal alignment around a shared vocabulary.
+ - **Vision:** To help folks choose and apply human centered design approaches for solving problems in government.
```

Comment:
Suggest changing the vision to read `To help folks use human-centered design to solve problems in government.`

Reply:
Yes, down with passive voice!